src/auspex/instruments/agilent.py

```diff
@@ -828,7 +828,7 @@ def get_trace(self, measurement=None):
         self.interface.write(":CALCulate:PARameter:SELect '{}'".format(measurement))
         self.reaverage()
         self.interface.write(":CALC:DATA? SDATA")
-        block =  self.interface.read_raw(size=256)
+        block =  self.interface.read_raw()
```

Comment:
We need to test this on other machines. @gribeill and I have had a miserable time getting VNA transfers working without a chunk limit on certain systems.